x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json

```diff
@@ -0,0 +1,853 @@
+              "title": "filters"
+            },
+            "gridData": {
+              "h": 5,
+              "i": "af1453d8-04d3-4b44-a3b0-138111255a23",
+              "w": 48,
+              "x": 0,
+              "y": 0
+            },
+            "panelIndex": "af1453d8-04d3-4b44-a3b0-138111255a23",
+            "panelRefName": "panel_0",
+            "title": "filters",
+            "version": "7.6.1"
+          },
+          {
+            "embeddableConfig": {
+              "title": "Bytes In"
+            },
+            "gridData": {
+              "h": 12,
+              "i": "14555108-559d-4c07-b240-6e6b14254f16",
+              "w": 24,
+              "x": 0,
+              "y": 5
+            },
+            "panelIndex": "14555108-559d-4c07-b240-6e6b14254f16",
+            "panelRefName": "panel_1",
+            "title": "Bytes In",
+            "version": "7.6.1"
+          },
+          {
+            "embeddableConfig": {
+              "title": "Packets In"
+            },
+            "gridData": {
+              "h": 12,
+              "i": "9c605367-60e3-4e9c-8036-a6191dbafe4a",
+              "w": 24,
+              "x": 24,
+              "y": 5
+            },
+            "panelIndex": "9c605367-60e3-4e9c-8036-a6191dbafe4a",
+            "panelRefName": "panel_2",
+            "title": "Packets In",
+            "version": "7.6.1"
+          },
+          {
+            "embeddableConfig": {
+              "title": "Bytes Out"
+            },
+            "gridData": {
+              "h": 12,
+              "i": "271558e6-b208-4e2c-abfb-0a6b2dbb0c66",
+              "w": 24,
+              "x": 0,
+              "y": 17
+            },
+            "panelIndex": "271558e6-b208-4e2c-abfb-0a6b2dbb0c66",
+            "panelRefName": "panel_3",
+            "title": "Bytes Out",
+            "version": "7.6.1"
+          },
+          {
+            "embeddableConfig": {
+              "title": "Packets Out"
+            },
+            "gridData": {
+              "h": 12,
+              "i": "41002ab1-845b-469e-9283-8a46a90e4662",
+              "w": 24,
+              "x": 24,
+              "y": 17
+            },
+            "panelIndex": "41002ab1-845b-469e-9283-8a46a90e4662",
+            "panelRefName": "panel_4",
+            "title": "Packets Out",
+            "version": "7.6.1"
+          },
+          {
+            "embeddableConfig": {
+              "title": "Bytes Dropped - no route"
+            },
+            "gridData": {
+              "h": 12,
+              "i": "b141f90b-739e-46f3-83c9-9c4661183837",
+              "w": 24,
+              "x": 0,
+              "y": 29
+            },
+            "panelIndex": "b141f90b-739e-46f3-83c9-9c4661183837",
+            "panelRefName": "panel_5",
+            "title": "Bytes Dropped - no route",
+            "version": "7.6.1"
+          },
+          {
+            "embeddableConfig": {
+              "title": "Packet Dropped - no route"
```

Comment:
s/Packet/Packets/

Reply:
@mtojek Good point, thanks for catching them. I just fixed them in the latest commit.